ovos_plugin_common_play/ocp/player.py

```diff
@@ -480,9 +480,9 @@ def play_next(self):
             return
         elif self.active_backend in [PlaybackType.SKILL,
                                      PlaybackType.UNDEFINED]:
-            # TODO: This is where Neon playback is failing
+            # TODO: Should UNDEFINED be handled as a skill?
```

Comment:
undefined should never happen, perhaps handle it differently? if undefined the best OCP can do is use the sync player GUI  (display only)

Reply:
I think it makes sense to deal with this separately; there should at least be a different logged error in this case